packages/checkout/widgets-lib/src/factory.ts

```diff
@@ -15,6 +15,7 @@ import { Sale } from 'widgets/sale/SaleWidgetRoot';
 import { Web3Provider } from '@ethersproject/providers';
 import { Bridge } from 'widgets/bridge/BridgeWidgetRoot';
 import { WalletConnectManager } from 'lib/walletConnect';
+import { ConnectConfig } from '@imtbl/checkout-sdk/dist/types';
```

Comment:
is it safe to get t his type from `dist`?

Reply:
Fixed.